model-engine/model_engine_server/infra/gateways/resources/k8s_endpoint_resource_delegate.py

```diff
@@ -1222,23 +1627,30 @@ async def _create_or_update_resources(
                 name=k8s_resource_group_name,
             )
 
-        pdb_config_arguments = get_endpoint_resource_arguments_from_request(
-            k8s_resource_group_name=k8s_resource_group_name,
-            request=request,
-            sqs_queue_name=sqs_queue_name_str,
-            sqs_queue_url=sqs_queue_url_str,
-            endpoint_resource_name="pod-disruption-budget",
-        )
-        pdb_template = load_k8s_yaml("pod-disruption-budget.yaml", pdb_config_arguments)
-        await self._create_pdb(
-            pdb=pdb_template,
-            name=k8s_resource_group_name,
-        )
+        if request.build_endpoint_request.nodes_per_worker == 1:
```

Comment:
I like what you did before with assigning a "is_multinode" var to `nodes_per_worker > 1`